change/@ni-nimble-components-30881da7-4a97-48ad-81c0-db79e1c01c98.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Adjust control height based on label presence",
+  "packageName": "@ni/nimble-components",
+  "email": "7282195+m-akinc@users.noreply.github.com",
+  "dependentChangeType": "patch"
+}
```

Comment:
As part of the bubble-up of the PR or as a follow-on, this workaround should be removed [from sl-properties](https://ni.visualstudio.com/DevCentral/_git/Skyline?path=/Web/Workspaces/SystemLinkShared/projects/systemlink-lib-angular/src/sl-properties/sl-properties.component.scss&version=GBmaster&line=36&lineEnd=37&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents)

Reply:
I will remove the workaround after these changes make it into systemlink-lib-angular.